### DISCUSSION TITLE
text recognition: is pretrained model on DICT90 available

### DISCUSSION THREAD

**Discussion by feiliu556** [open-mmlab/mmocr#1847] [Q&A]
hello all, 
I am wondering if the pretrained models on DICT90 or DICT91 are available?
It seems that only the models pretrained on DICT37 for CRNN and ABINet are provided.

**Comment by gaotongxiao** [comment]
Answering here again just for record: do a global search for "english_digits_symbols.txt" and you should be able to see the models trained with DICT90.
